ntbs-service/DataAccess/AlertRepository.cs

```diff
@@ -44,9 +44,12 @@ public async Task<Alert> GetAlertByNotificationIdAndTypeAsync(int? notificationI
                 .SingleOrDefaultAsync(m => m.NotificationId == notificationId && m.AlertType == alertType);
         }
 
-        public async Task<IList<Alert>> GetAlertsByTbServiceCodesAsync(IEnumerable<string> tbServices)
+        public async Task<IList<Alert>> GetDateOrderedAlertsByTbServiceCodesAsync(IEnumerable<string> tbServices)

```

Comment:
It's a little intuitive to have the `Date` be the first word after get, as it makes its seem like we're getting some dates.
Do the consumers need to know that it's ordered?